config.toml

```diff
@@ -13,6 +13,7 @@ font_awesome_version = "5.3.1"
 
 summary = """
 Flux is a tool that automatically ensures that the state of your [Kubernetes](https://kubernetes.io) cluster matches the configuration you've supplied in Git. It uses an [operator](https://github.com/operator-framework/awesome-operators) in the cluster to trigger deployments *inside* Kubernetes, which means that you don't need a separate continuous delivery tool.
+📣 Flux is in maintenance mode and the we will soon guide the community to migrate from Flux v1 to the new architecture (Flux v2 and the GitOps Toolkit). [Learn more](https://toolkit.fluxcd.io/) and [join the discussion!](https://github.com/fluxcd/toolkit/discussions)
```

Comment:
```suggestion

📣 [Flux v1 is in maintenance mode](https://github.com/fluxcd/flux/issues/3320)! We will soon start publishing guides which help you migrate Flux v2 and the GitOps Toolkit.

[Learn more](https://toolkit.fluxcd.io/) and [join the discussion!](https://github.com/fluxcd/toolkit/discussions)
```

Reply:
This change adds line breaks to make things a bit more obvious, also adds a link to the issue about maintenance mode for more background.

Reply:
![Screenshot from 2020-10-06 09-34-25](https://user-images.githubusercontent.com/1346979/95172143-31274980-07b7-11eb-8ee8-e78e79eb5ac6.png)